packages/amplify-e2e-tests/src/__tests__/rds-mysql-auth-apikey-lambda.test.ts

```diff
@@ -0,0 +1,449 @@
+import {
+  addApiWithAllAuthModes,
+  amplifyPush,
+  createNewProjectDir,
+  deleteDBInstance,
+  deleteProject,
+  deleteProjectDir,
+  getAppSyncApi,
+  getProjectMeta,
+  importRDSDatabase,
+  initJSProjectWithProfile,
+  setupRDSInstanceAndData,
+  sleep,
+} from 'amplify-category-api-e2e-core';
+import { existsSync, writeFileSync } from 'fs-extra';
+import generator from 'generate-password';
+import path from 'path';
+import { GQLQueryHelper } from '../query-utils/gql-helper';
+import { getConfiguredAppsyncClientAPIKeyAuth, getConfiguredAppsyncClientLambdaAuth } from '../schema-api-directives';
+
+// to deal with bug in cognito-identity-js
+(global as any).fetch = require('node-fetch');
+
+describe('RDS Relational Directives', () => {
+  const [db_user, db_password, db_identifier] = generator.generateMultiple(3);
+
+  // Generate settings for RDS instance
+  const username = db_user;
+  const password = db_password;
+  let region = 'ap-northeast-2';
+  let port = 3306;
+  const database = 'default_db';
+  let host = 'localhost';
+  const identifier = `integtest${db_identifier}`;
+  const projName = 'rdsmodelauthtest';
+
+  let projRoot;
+  let blogApiKeyClient, postApiKeyClient, userApiKeyClient, profileApiKeyClient;
+  let blogLambdaClient, postLambdaClient, userLambdaClient, profileLambdaClient;
+
+  beforeAll(async () => {
+    projRoot = await createNewProjectDir('rdsmodelapi');
+    await initProjectAndImportSchema();
+    await amplifyPush(projRoot);
+    await sleep(2 * 60 * 1000); // Wait for 2 minutes for the VPC endpoints to be live.
+
+    const meta = getProjectMeta(projRoot);
+    const appRegion = meta.providers.awscloudformation.Region;
+    const { output } = meta.api.rdsmodelauth;
+    const { GraphQLAPIIdOutput, GraphQLAPIEndpointOutput, GraphQLAPIKeyOutput } = output;
+    const { graphqlApi } = await getAppSyncApi(GraphQLAPIIdOutput, appRegion);
+
+    expect(GraphQLAPIIdOutput).toBeDefined();
+    expect(GraphQLAPIEndpointOutput).toBeDefined();
+    expect(GraphQLAPIKeyOutput).toBeDefined();
+
+    console.log(`Url: ${GraphQLAPIEndpointOutput}`);
+    console.log(`API Key: ${GraphQLAPIKeyOutput}`);
```

Comment:
don't need these

Reply:
Few things left out from my troubleshooting, removing the console logs.